enterprise/cypher/acceptance-spec-suite/src/test/scala/org/neo4j/internal/cypher/acceptance/NodeIndexSeekByRangeAcceptanceTest.scala

```diff
@@ -928,19 +926,14 @@ class NodeIndexSeekByRangeAcceptanceTest extends ExecutionEngineFunSuite with Cy
 
     val query = "MATCH (n:Label) WHERE n.prop >= '1' AND n.prop > 10 RETURN n.prop AS prop"
 
-    an[IllegalArgumentException] should be thrownBy {
```

Comment:
Is that `IllegalArgumentException` not thrown anymore or do we not assert it is thrown any more?

Reply:
this was wrongly removed